SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -503,7 +504,10 @@ void UGlobalStateManager::AuthorityChanged(const Worker_AuthorityChangeOp& AuthO
 			// that was authoritative over the GSM restarts.
 			if (!bCanBeginPlay)
 			{
-				BecomeAuthoritativeOverAllActors();
+				if (!GetDefault<USpatialGDKSettings>()->bEnableUnrealLoadBalancer)
+				{
+					BecomeAuthoritativeOverAllActors();
+				}
```

Comment:
SetCanBeginPlayer is also called via the ServerTravel code - AFAIK this will all break in multi-worker - perhaps add a TODO/check etc. that that code isn't hit when the LB is on?

Reply:
server travel "does not work in zoned" - matt